### DISCUSSION TITLE
2024/10/03/DAC-Giclee-Photography-Output/

### DISCUSSION THREAD

**Discussion by giscus** [MrWillCom/MrWillCom.github.io#13] [Announcements]
# 2024/10/03/DAC-Giclee-Photography-Output/

上海高中每年秋天都会有明信片互寄的活动，同学们只需要写好明信片投进相应的小箱子里就行了，而外联部要考虑的就多了。很多社团还有少数个人会在这时候卖明信片。大家会在大群里发问卷和收款码，以一种高度文明的方式做生意。前年看到后，就决定第二年自己也整点。于是去年十月份整理了一些喜欢的照片，做成了明信片。  这属于是我第一次以实体的形式输出照片。把自己的明信片寄给别人的感觉真的很不错。😋 后来在 b 站上

https://blog.mrwillcom.com/2024/10/03/DAC-Giclee-Photography-Output/

**Comment by JimmyHuang037** [comment]
nb！
